File: homafo_backend/src/main/java/com/bioagricola/aforos/entity/dto/DsusSuscripcionResource.java
package com.bioagricola.aforos.entity.dto;

import java.math.BigDecimal;
import java.sql.Timestamp;
import java.util.Date;


import lombok.Data;

@Data
public class DsusSuscripcionResource {

	private Long dsusIderegistr;
	private String dsusEstado;
	private String dsusDescripcion;
	private String dsusPcodigo;
	private Long susIderegistro;
	private Long terIderegistro;
	private TerTerceroResource terTerceroResource;
	private Long proIderegistro;
	private Long uniMunicipio;
	private Long estTipsuscripc;
	private Long uniTipsuscripc;
	private Integer estTipusosuscr;
	private Long uniTipusosuscr;
	private Integer empIderegistro;
	private Long estLiquidacion;
	private Long uniLiquidacion;
	private Long cicIderegistro;
	private Date dsusFecinicio;
	private Date dsusFecexpira;
	private Integer proCatestrato;
	private Date dsusIniestado;
	private Date dsusFinestado;
	private BigDecimal dsusFactor;
	private Long usuIderegistro;
	private Long uniActsuscripc;
	private String dsusResolestrato;
	private Date dsusFecact;
	private String terceroNombreCompleto;
	private String terceroDocumento;

}

File: homafo_frontend/src/models/dto/DsusSuscripcionResource.ts
import TerTerceroResource from "./TerTerceroResource";

export default interface DsusSuscripcionResource {

    dsusIderegistr:number;
	dsusEstado:String;
	dsusDescripcion:String;
	dsusPcodigo:String;
	susIderegistro:number;
	terIderegistro:number;
	terTerceroResource:TerTerceroResource[];
	proIderegistro:number;
	uniMunicipio:number;
	estTipsuscripc:number;
	uniTipsuscripc:number;
	estTipusosuscr:number;
	uniTipusosuscr:number;
	empIderegistro:number;
	estLiquidacion:number;
	uniLiquidacion:number;
	cicIderegistro:number;
	dsusFecinicio:Date;
	dsusFecexpira:Date;
	proCatestrato:number;
	dsusIniestado:Date;
	dsusFinestado:Date;
	dsusFactor:number;
	usuIderegistro:number;
	uniActsuscripc:number;
	dsusResolestrato:String;
	dsusFecact:Date;
	terceroNombreCompleto:string;
	terceroDocumento:string;
}